parse_results.php

```diff
@@ -99,8 +114,14 @@ function parse_results($path) {
         $project['phpunit']['assertions'] = (int) $phpunit->testsuite['assertions'];
         $project['phpunit']['failures'] = (int) $phpunit->testsuite['failures'];
         $project['phpunit']['errors'] = (int) $phpunit->testsuite['errrors'];
+        if ($project['phpunit']['failures'] > 0 || $project['phpunit']['errors'] > 0) {
+          $project['is_failed']['failed'] = TRUE;
```

Comment:
The structure is strange, you have an array, but only have a single key inside that you set to TRUE and then loop over it. Why not just set $project['is_failed'] = TRUE, with a default value of FALSE, similar to how you handle changed?